packages/fxa-customs-server/lib/config/config.js

```diff
@@ -424,7 +424,7 @@ module.exports = function (fs, path, url, convict) {
       totpCodeRules: {
         actions: {
           doc: 'Array of actions that this rule should be applied to',
-          default: ['verifyTotpCode'],
+          default: ['verifyTotpCode', 'verifyRecoveryPhoneTotpCode'],
```

Comment:
I wonder if this should be its own entry? The unconfirmed codes are valid for [10mins](https://github.com/mozilla/fxa/blob/56beb1238a5b1ab36bbf30e425ef54c05f62ae55/libs/accounts/recovery-phone/src/lib/recovery-phone.manager.ts#L24).

This rule says the user can attempt 2 times in 30seconds before rate limited for 30seconds, which makes sense for authenticator totp based codes.

I don't recall what the rate limit was decided confirming codes but maybe 10 attempts every 15mins is ok? Then rate limit 10mins for the code to expire?